bip-0141.mediawiki

```diff
@@ -116,6 +116,16 @@ Blocks are currently limited to 1,000,000 bytes (1MB) total size. We change this
 
 The new rule is ''block weight'' ≤ 4,000,000.
 
+===== Transaction size calculations =====
+
```

Comment:
Not sure what you mean here. Transaction weight and Transaction virtual size *are* used in the code, and transaction weight is enforced in IsStandardTx() policy.